universe/manifest.tt

```diff
@@ -0,0 +1,76 @@
+class @vikram.hello
+#_[thingpedia_name="Test 1"]
+#_[thingpedia_description="Test 1 for Movies"]
+#[license="BSD-3-Clause"]
+#[license_gplcompatible=true]
+#[subcategory="media"]
+{
+    import loader from @org.thingpedia.v2();
+    import config from @org.thingpedia.config.none();
+    import config from @org.thingpedia.config.none(api_key=$?);
```

Comment:
You duplicated the "import config" line. You only need one - that's why you're getting the error.